frontend/src/components/CustomerStakesVisualization.tsx

```diff
@@ -11,7 +11,6 @@ import colors from '../colors.module.scss';
 const classes = classNames.bind(css);
 
 const noRatingsStakes = [
-  { id: 1, value: 0.5 },
   { id: 2, value: 0.3 },
   { id: 3, value: 0.1 },
   { id: 4, value: 0.1 },
```

Comment:
Would it be better if the values would get gradually lower? Currently the id: 2 is the biggest dot, and 3 & 4 are the same size.

In my opinion something like:

```
const noRatingsStakes = [
  { id: 2, value: 0.3 },
  { id: 3, value: 0.25 },
  { id: 4, value: 0.1 },
];
```

looked a bit better. What do you think?

Reply:
I submitted one version with the second value of 0.23 (I felt 0.25 was too large).
Overall I thought both look good, so I sent a question to Minna and your suggestion is now also design-approved.

Reply:
Looking good 👍